django/db/models/fields/__init__.py

```diff
@@ -1085,6 +1085,8 @@ def formfield(self, **kwargs):
         # will be validated twice. This is considered acceptable since we want
         # the value in the form field (to pass into widget for example).
         defaults = {'max_length': self.max_length}
+        if self.null and not connection.features.interprets_empty_strings_as_nulls:
```

Comment:
Ok, maybe that is a larger issue we should tackle separately. I'd still suggest at least a comment to make it easier to find some of these locations later, if someone ever decides to fix heterogenous multi-db setups.